colpali-trial.py

```diff
@@ -0,0 +1,49 @@
+from typing import List
```

Comment:
do we need this file? AFAIU, looks like experimentation. Maybe we can have a .gitignore experimentation folder.